docs/api/frameless-window.md

```diff
@@ -18,14 +18,26 @@ const win = new BrowserWindow({ width: 800, height: 600, frame: false })
 win.show()
 ```
 
-### Alternatives on macOS
+### Alternatives
 
 There's an alternative way to specify a chromeless window.
 Instead of setting `frame` to `false` which disables both the titlebar and window controls,
 you may want to have the title bar hidden and your content extend to the full window size,
-yet still preserve the window controls ("traffic lights") for standard window actions.
+yet still preserve the window controls ("traffic lights" on macOS) for standard window actions.
 You can do so by specifying the `titleBarStyle` option:
 
+### Alternatives on macOS
```

Comment:
I think this header is unneeded as overlay will be available on both Windows and macOS.

Reply:
The options `hidden`, `hiddenInset`, and `customButtonsOnHover` will still only be on macOS though, correct? I think this edit just got applied to the wrong line and should be after the `overlay` option example instead.

Reply:
oh right.  yes.